src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1049 @@
+'use strict';
+
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['PropTypes' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5603033)